### PR TITLE
Cleanup spaces in config macros

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

File: addons/ace_interact/ace_arsenal_stats.hpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
     class statBase;
     class GVAR(frequencyRange): statBase {
         scope = 2;
-        stats[] = {QEGVAR(arsenalStats,frequencyMin), QEGVAR(arsenalStats,frequencyMax)};
+        stats[] = {QEGVAR(arsenalStats,frequencyMin),QEGVAR(arsenalStats,frequencyMax)};
```
Comment (jonpas) [inline addons/ace_interact/ace_arsenal_stats.hpp]: This space was not in a macro.

**Comment by BrettMayson** [comment]
You can have spaces in the define, those are fine.

You just can't have them in the call, that's where they get kept

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
@PabstMirror Do you have a shell script or something you did this with? It needs to be applied to more files with newer HEMTT (radio states, menus, etc.).

**Comment by PabstMirror** [comment]
https://github.com/IDI-Systems/acre2/pull/1304 should get it building, macro space should just be a warning

**Comment by jonpas** [comment]
#1317 already made it build, I didn't see that PR of yours. I'd like to cleanup those spaces as well though.

**Pull request merged**